examples/iso/k3s-single-node.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: 1.0
+image:
+  arch: x86_64
+  imageType: iso
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso
+  outputImageName: single-node-k3s-no-selinux.iso
+kubernetes:
+  version: v1.28.7+k3s1
```

The supported version is 1.28.8 in Edge 3.0.0:
```suggestion
  version: v1.28.8+k3s1
```